packages/flame_behaviors/example/analysis_options.yaml

```diff
@@ -0,0 +1,5 @@
+include: package:flame_lint/analysis_options_with_dcm.yaml
+
+linter:
+  rules:
+    public_member_api_docs: false
```

Comment:
There shouldn't be any public facing API in the example?

Reply:
See comment above https://github.com/flame-engine/flame/pull/3717#discussion_r2405924922